CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+## [3.0.0-beta.11] - 2021-03-11
+
 Breaking changes:
```

Comment:
Pour garder la même organisation que sur les releases beta précédentes :

```
Breaking changes:

- Delete the following deprecated components:
  - `Paragraph` (replaced by `Next` `Paragraph`)
- Delete the following deprecated SCSS mixins and styles:
  - `k-Title`
  - `k-Paragraph`
- Remplace prop `margin` with `noMargin` on `Title`.
- Remplace prop `margin` with `noMargin` on `Paragraph`.

Features:

- Convert the following component to CSS-in-JS:
  - `Title`
```

(je ne suis plus sûr si ce que j'ai mis pour le remplacement de Paragraph est conforme à ce qui a été fait)

Reply:
@joachimesque je vais refaire une PR de release en prenant toutes les PR dans " In next beta"